### PR TITLE
Endpoints to operate with files for sandbox

### PR DESCRIPTION
**Pull request by nulls**
It's related to #1274

### CODE REVIEW

File: save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt [hunk 1]
```diff
@@ -35,11 +35,22 @@ import javax.transaction.Transactional
...
-    val storage: Storage<SandboxStorageKey>,
+    val storage: SandboxStorage,
     val sandboxExecutionRepository: SandboxExecutionRepository,
     val sandboxUserRepository: SandboxUserRepository,
     val agentsController: AgentsController,
 ) {
+    /**
+     * @param userName
+     * @return list of available files for provided [userName]
+     */
```
Comment (sanyavertolet) [inline save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt:46]: Shouldn't we use SwaggerDoc here?

**Review by sanyavertolet — APPROVED** [review]

File: save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt (hunk 1 above)
Comment (nulls) [reply save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt:46]: Probably we can add it for current controller

Comment (sanyavertolet) [reply save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt:46]: Will you do it?🥺

Comment (nulls) [reply save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt:46]: doing it right now

**Review by sanyavertolet — APPROVED** [review]

**Pull request closed without merging**